evalml/pipelines/utils.py

```diff
@@ -130,7 +131,9 @@ def _get_preprocessing_components(
 
     # The URL and EmailAddress Featurizers will create categorical columns
     categorical_cols = list(
-        X.ww.select(["category", "URL", "EmailAddress"], return_schema=True).columns
+        X.ww.select(
+            ["category", "URL", "EmailAddress", "IPAddress"], return_schema=True
```

Comment:
@chukarsten How come we need to make this change? AutoMLSearch and our pipelines can't handle the `IPAddress` logical type yet.